### PR TITLE
New packages: klfc-1.5.7 & ascii-image-converter-1.11.0

### PR DESCRIPTION
**Pull request by nadevko**
#### Testing the changes

- I tested the changes in this PR: **YES**

#### New package

- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing

- I built this PR locally for my native architecture, **x86_64-glibc**

### CODE REVIEW

File: srcpkgs/ascii-image-converter/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+version='1.11.0'
+checksum='17f87445971fdb491c6ca9d4beed67044a330339b47775adffb8e3cb898ec15d'
+
+distfiles="https://github.com/TheZoraiz/ascii-image-converter/releases/download/v$version/ascii-image-converter_Linux_amd64_64bit.tar.gz>v$version.tar.gz"
+nopie='/usr/bin/ascii-image-converter'
+
+do_extract() {
+        bsdtar --strip-components 1 -xf $XBPS_SRCDISTDIR/${wrksrc##*/}/v$version.tar.gz
+}
```
Comment (classabbyamp) [inline srcpkgs/ascii-image-converter/template:17]: should not be necessary, xbps-src knows how to unpack tar.gz and zip files itself

File: srcpkgs/ascii-image-converter/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+short_desc="Tool to convert images into ascii art and braille art"
+maintainer='Nadeŭka <nadevko@riseup.net>'
+license='Apache-2.0'
+homepage='https://github.com/TheZoraiz/ascii-image-converter'
+revision='1'
+
+version='1.11.0'
+checksum='17f87445971fdb491c6ca9d4beed67044a330339b47775adffb8e3cb898ec15d'
+
+distfiles="https://github.com/TheZoraiz/ascii-image-converter/releases/download/v$version/ascii-image-converter_Linux_amd64_64bit.tar.gz>v$version.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/ascii-image-converter/template:12]: should not be necessary to change the filename

**Review by classabbyamp — CHANGES_REQUESTED** [review]
templates should build from source, not just download the binaries from upstream. This will not work for cross, or anything other than x86_64 glibc, as-is.

Please also run `xlint` (from `xtools`) on the templates

**Comment by classabbyamp** [comment]
also, unless one of these packages depends on the other, they should be in separate PRs

**Comment by nadevko** [comment]
Thanks, I'll fix it tomorrow,

**Pull request closed without merging**